AutomatedTesting/src/test/resources/features/DeletePlanet.feature

```diff
@@ -0,0 +1,11 @@
+Feature: DeletePlanet
+
+    Scenario: Automated Delete Planet Test Case
+    Validating that Planets can be deleted by users no matter if the Planets have any orbiting Moons or not.
```

Comment:
I would be more descriptive in this Validating... statement. You can use multiple sentences and it should be clear about all the requirements for different inputs. Not only should it work if Planets have moons or not but also what is the result if there are moons that orbit the planet.